python/tskit/trees.py

```diff
@@ -4886,7 +4893,7 @@ def genotype_matrix(
             isolated_as_missing=isolated_as_missing, alleles=alleles
         )
 
-    def alignments(self, *, reference_sequence=None):
+    def alignments(self, *, reference_sequence=None, missing_data_character=None):
         """
         Returns an iterator over the full sequence alignments for the samples in this
         tree sequence. Each alignment ``a`` is a string of length ``L`` where ``L``
```

Comment:
Perhaps "``L`` is the sequence length of this tree sequence (which must have :ref:`discrete_genome` equal to ``True``), and ...

Reply:
Done.